praw/exceptions.py

```diff
@@ -172,6 +172,17 @@ def __init__(self):
         )
 
 
+class InvalidMedia(ClientException):
```

Comment:
I feel like this should inherit from `APIException` or `RedditAPIException` because a `ClientException` is for "exceptions that don't involve interaction with Reddit's API." However, I realize that those two have initializers that expect certain parameters to be there that won't be there in this case. Do you have a good idea of a solution to this conundrum?